standalone/instructlab-rhoai.md

```diff
@@ -0,0 +1,320 @@
+    AWS_BUCKET_NAME=<bucket-name> \
+    S3_FOLDER=prometheus-eval \
+    MODEL_PATH=model \
+    HF_TOKEN=<hugging-face-auth-token>
+    ```
+
+* Navigate to the OpenShift AI dashboard
+    * Choose Data Science Projects from the left hand menu and choose your data science project/namespace.
+    * Choose the data connections tab, and click on the Add data connection button. Enter the details of your S3 bucket (object store) and click Add data connection.
+
+    `Note`: Before following the next step - Ensure that the CapabilityServiceMeshAuthorization status is True in DSCinitialization resource.
+
+    * Create a model server instance
+        * Navigate to Data Science Projects and then the Models tab
+        * On the right hand side select ‘Deploy model’ under Single-model serving platform
+        * Under Serving runtime choose the serving runtime `vLLM Serving Runtime for Kserve`.
+        * Check the `Make deployed models available through an external route` box.
+        * Under token authentication check the `Require token authentication` box, select the service account that we have created above.
+        * Choose the existing data connection created earlier.
+        * Click deploy.
+
+* Create a secret containing the judge model serving details
+
+    ```
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: judge-serving-details
+    type: Opaque
+    stringData:
+      JUDGE_NAME:               # Name of the judge model or deployment
+      JUDGE_ENDPOINT:           # Model serving endpoint, Sample format - `https://<deployed-model-server-endpoint>/v1`
+      JUDGE_API_KEY:            # Deployed model-server auth token
+      JUDGE_CA_CERT:            # Configmap containing CA cert for the judge model (optional - required if using custom CA cert), Example - `kube-root-ca.crt`
+      JUDGE_CA_CERT_CM_KEY:     # Name of key inside configmap (optional - required if using custom CA cert), Example - `ca.crt`
+    ```
+
+* If using a custom CA certificate you must provide the relevant data in a ConfigMap. The config map name and key are then provided as a parameter to the standalone.py script as well as in the `judge-serving-details` secret above.
+
+### Step 3 (Optional) - Create NFS Server
+
+  This step is needed when the cluster doesn't have a storage provisioner capable of provisioning PersistentVolumeClaim with ReadWriteMany capability.
+
+**Create NFS Server using the CSI Driver**
+
+  * Installing the NFS CSI driver
+    ```
+    $ curl -skSL https://raw.githubusercontent.com/kubernetes-csi/csi-driver-nfs/v4.9.0/deploy/install-driver.sh | bash -s v4.9.0 --`
+    ```
+
+  * For deploying a in-cluster NFS server
+    ```
+    oc apply -f ./nfs-server-deployment.yaml
+    oc apply -f ./nfs-storage-class.yaml
+    ```
+
+  * This will create the required resources in the cluster, including the required StorageClass.
+
+
+## Now we can continue to set up the required resources in our cluster:
```

Comment:
I'd suggest to call that part "Step 4 - Run InstructLab distributed training" and do not number the included steps to keep the logical structure of the process simple.